src/main/java/webApp/QuestionMessage.java

```diff
@@ -20,14 +23,16 @@
      * A numeric based question message constructor with boundaries
      * @param type  the question type
      * @param question  the question text
+     * @param options  the options to display to the user for the question
      * @param min   the minimum value
      * @param max   the maximum value
      * @param stringAnswer   the answer if the type is string
      * @param numberAnswer   the answer if the type is numerical
      */
-    public QuestionMessage(String type, String question, int min, int max, String stringAnswer, int numberAnswer, Collection<String> stringAnswerList, Collection<Integer> numberAnswerList) {
+    public QuestionMessage(String type, String question, Collection<String> options, int min, int max, String stringAnswer, int numberAnswer, Collection<String> stringAnswerList, Collection<Integer> numberAnswerList) {
```

Comment:
We really should figure out a better way of going about this at some point, oh and my unit test involving this class will need to be updated to accommodate this change. It's under QuestionMessageTest.java. It will break with this change.

Reply:
For sure agree. And yeah as you mentioned, tests have been fixed.